dist/tools/ci/static_tests.sh

```diff
@@ -110,7 +110,7 @@ DIFFFILTER="AC" run ./dist/tools/licenses/check.sh
 run ./dist/tools/doccheck/check.sh
 run ./dist/tools/externc/check.sh
 run ./dist/tools/cppcheck/check.sh
-run ./dist/tools/vera++/check.sh
+# run ./dist/tools/vera++/check.sh
```

Comment:
should it be like that?
At least add a comment why it's disabled, if not just remove it.